### ISSUE TITLE
Can't open fits

### ISSUE THREAD

**Issue by stevenstetzler** [gmegh/fitsimagevoy#1]
I'm trying to open a fits file in VS Code and it gives this error:

```
Assertion Failed: argument is undefined or null
```

It opens okay in DS9. Any idea what could be happening? Here's the file: http://epyc.astro.washington.edu/~stevengs/891840.fits

VS Code version info:
```
Version: 1.85.2
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:11.430Z (4 wks ago)
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 21.6.0
```

**Comment by gmegh** [comment]
Hi @stevenstetzler ! Thanks for reaching out. A couple of questions: Is python available in the environment where you are running VSCode? Were you able to open other files? 
I am aware that if python is not available, it can't open the file. I have to add a warning for that case, but it could be there is another issue going on here, so I am eager to hear about it and try to fix it!

**Comment by stevenstetzler** [comment]
It looks like python isn't available in my base environment. If I make any version of python available, will that allow the extension to work?

**Comment by stevenstetzler** [comment]
Also do you know how to make python available to VS Code on a remote machine? It looks like it's not enough to set `PATH` in `~/.bashrc` or `~/.bash_profile`.

**Comment by gmegh** [comment]
Yes, if you make python available it should work. 
Is there any pre-set you have to do to activate python? For instance, for Rubin/LSST remote machines, python is only available on the machine once you activate the science pipelines. If that is the same case for you, I can add a setting that is a n initial command for the extension to activate python in the environment.

**Comment by stevenstetzler** [comment]
Yes, typically I need to make python (or at least a recent version of Python) available by activating a conda environment or something similar. An init command might be the way to go.

**Comment by gmegh** [comment]
That sounds right. I have a branch that allows for init commands, but I
need to add it so that it can be edited from the extension settings.

Were you able to use the extension on your local? Any feedback would be
appreciated!

**Comment by stevenstetzler** [comment]
It doesn't look like the extension works for me locally either, on Mac OS. Are there any logs I can inspect to see what's going on? It is the same error:
```
Assertion Failed: argument is undefined or null
```

**Comment by tburnett** [comment]
I get the same, Windows machine., remote to WSL, a very standard FITS file.
But also:  on a remote to another machine, an empty setup screen

**Comment by JakeDolbit** [comment]
Same error message.
VSCode version info:

```
Version: 1.91.1 (user setup)
Commit: f1e16e1e6214d7c44d078b1f0607b2388f29d729
Date: 2024-07-09T22:06:49.809Z
Electron: 29.4.0
ElectronBuildId: 9728852
Chromium: 122.0.6261.156
Node.js: 20.9.0
V8: 12.2.281.27-electron.0
OS: Windows_NT x64 10.0.22631
```

I tried also on Ubuntu instead of Windows, there it works. Same version info as above, but
`OS: Linux x64 6.5.0-41-generic`

**Comment by tburnett** [comment]
My primary use case is to check FITS files with multiple table extensions.  
It does now give header info.  (v0.0.5). But the text is not readable in a dark color theme.
Files with HEALPix-format tables do not display the corresponding all-sky image, which would be nice to see.
Some files have a ".fit" extension. How would I tell VScode about that?

**Comment by tburnett** [comment]
The HDU pull-down menu has the index only. Could it also show the EXTNAME?

**Comment by Bhavesh012** [comment]
I have the same issue.

`Assertion Failed: argument is undefined or null`

Python is installed in the base environment with all the required packages.
